scripts/upgrade

```diff
@@ -128,17 +128,8 @@ ynh_system_user_create $app # Create dedicated user if not existing
 # We store photos (potentially large data) on /home/yunohost.app
 datapath=/home/yunohost.app/$app
 
-# Backward compatibility:
-# If the galleries subdirectory was moved,
-# remove the link and overwrite it
-# (this directory always includes only a single index.php file...)
```

Comment:
If the directory `galleries` contains an index.php, shouldn't you add this file from the original directory ?
During the install, you seems to only create an empty directory. Is it enough ?

Reply:
You're right, we shouldn't have to ask ourselves the questions :smile: 
Should be fixed.